framework/doc/content/source/transfers/MultiAppInterpolationTransfer.md

```diff
@@ -1,8 +1,25 @@
 # MultiAppInterpolationTransfer
 
-The MultiAppInterpolationTransfer transfers the source variable to the nearest node on the
+The `MultiAppInterpolationTransfer` transfers the source variable to the nearest node on the
 target mesh using mesh interpolation, including the ability to utilize the displaced
-configuration for either or both the source and target.
+configuration for either or both the source and target.  
+
+Nodal transfers using the default settings for this interpolation scheme, `interp_type=inverse_distance` and `num_points=3`, are 
+best suited for the interpolation of point cloud data in the source mesh onto a target mesh.  
+The [MultiAppMeshFunctionTransfer](/MultiAppMeshFunctionTransfer.md) may be a better choice for nodal transfers 
+between two meshes because the element shape functions will be used in the transfer.
```

Comment:
MultiAppMeshFunctionTransfer does not offer extrapolation, and MultiAppInterpolationTranser offers  extrapolation. That is, a target point value can be computed using the nearest source nodes when a target point is not inside of the source domain.

Reply:
@lynnmunday can you move this to a "see also" section and address Fande's comment?

Reply:
actually you mention the non-extrapolation nature above. this is a bit disjointed